js/ui/map.js

```diff
@@ -575,7 +575,7 @@ class Map extends Camera {
      * representing features within the specified vector tile or GeoJSON source that satisfy the query parameters.
      *
      * @param {string} sourceID The ID of the vector tile or GeoJSON source to query.
-     * @param {Object} parameters
+     * @param {Object} [parameters]
```

Comment:
Got it. This is more of a feature addition than a bug fix. We never supported or claimed to support omitting `parameters`. 👍